libs/platform/src/lib/table/table.component.scss

```diff
@@ -141,8 +141,17 @@ fdk-dynamic-portal {
         border-spacing: 0;
     }
 
-    /** Temporary until arrow keyboard navigation not implemented */
-    .fd-table__body:focus {
+    .fd-table__body__focus-mock {
```

Comment:
Looks great. Is it a -mock or more a -wrapper?

Reply:
It's a mock